src/plugins/intel_gpu/src/graph/graph_optimizer/reorder_inputs.cpp

```diff
@@ -740,6 +741,29 @@ void reorder_inputs::run(program& p, reorder_factory& rf) {
         }
     };
 
+    const auto reorder_input_group_normalization = [&p, &rf](typed_program_node<group_normalization>& group_normalization_node) {
+        // Change scale and bias formats of group_normalization node from fsv16 to bfyx
+        auto scale_dep = group_normalization_node.get_dependency_with_port(1);
+        auto bias_dep = group_normalization_node.get_dependency_with_port(2);
+        const auto& scale = scale_dep.first;
+        const auto& bias = bias_dep.first;
+        auto scale_layout = scale->get_output_layout();
+        auto bias_layout = bias->get_output_layout();
+        if (scale_layout.format == format::b_fs_yx_fsv16) {
+            auto new_scale_layout = scale_layout;
+            auto new_bias_layout = bias_layout;
+            new_scale_layout.format = format::bfyx;
+            new_bias_layout.format = format::bfyx;
+            auto scale_input = rf.get_reorder(scale->id(), scale_dep.second, scale_layout, new_scale_layout);
+            auto bias_input = rf.get_reorder(bias->id(), bias_dep.second, bias_layout, new_bias_layout);
+            if (scale_input.first && bias_input.first) {
+                p.add_intermediate(scale_input.first, group_normalization_node, 1);
+                p.add_intermediate(bias_input.first, group_normalization_node, 2);
+                group_normalization_node.recalc_output_layouts();
+            }
+        }
+    };
```

Comment:
If this is a required format of `group_normalization`, what about implementing it in `query_formats`?

Reply:
1. This only works when the kernel supports bfyx input and fsv output 
2. Also the expectation is to have get_target_input_format in the reorder_inputs for the GN if you set above for GN. You can check whether it is set as intended. 
3. BTW chainging the format does not affect perforamnce? I tought better option is not to fix the foramt but to fix the shape as expected

Reply:
1. The kernel works with both the bfyx and fsv input formats, and it was checked through unit tests. Since the kernel processes the values sequentially, it seems to support both formats.
2. While attempting to apply get_target_input_format, it seemed that the function is typically used to traverse predecessors or successors. In the case of GN,  which only involves scale and bias, would it still be appropriate to use get_target_input_format in this case?
   I'm not sure I fully understand the phrase "You can check whether it is set as intended." Could you explain it in more detail?
3. Since the values are processed sequentially and the inserted reorders are optimized, no significant impact on performance is expected.

Reply:
// reponse to 2
    auto ret = node->get_preferred_output_fmt(user_idx); won't be format::any if you implement query_format. 

// response to 3
 - How can we ensure "no significant impact on perf expected" with a PR that changes format?  :)
 - I recommend to run a benchmark to check a performance

Reply:
I meant for  benchmark as to trigger the performance benchmark jenkins.

Reply:
And please move this to query_input if you are intending to add reorder as Mingyu mentioned.

Reply:
The fix was move to query_formats as you and @isanghao suggested.
Thanks for your comments.
The performance will be evaluated as soon as the DLBenchmark server becomes available.

Reply:
After discussing this fix with Taylor, query_formats seemed to be used for this issue.
Thanks for your comment.